docs/endpoints/signups.md

```diff
@@ -10,6 +10,8 @@ POST /api/v3/signups
     The drupal node id of the campaign the user is signing up for.
   - **campaign_run_id**: (int) optional.
     The drupal campaign run node id of the campaign run the user is signing up for.
+  - **northstar_id**: (int) optional.
```

Comment:
Do we need to be clear about the requirements making this request `asAUser` vs `asClient` because `northstar_id` is only optional when acting as a client right?

Reply:
@sbsmith86 just updated this! Does this make more sense?

Reply:
yes! amazing